core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -255,12 +255,12 @@ object KafkaConfig {
   val LogFlushIntervalMsProp = "log.flush.interval.ms"
   val LogFlushOffsetCheckpointIntervalMsProp = "log.flush.offset.checkpoint.interval.ms"
   val LogPreAllocateProp = "log.preallocate"
-  val MessageFormatVersionProp = "message.format.version"
+  val LogMessageFormatVersionProp = "log.message.format.version"
```

Comment:
Do you think we can just reuse the definition in LogConfig and append "log."? This will make the relationship clearer and prevent accidental diversion.

sorry for nitpick, but after seeing all the places where same config has entirely different name in Log and Kafka, I'm a bit concerned.